src/bin/todo/completion.rs

```diff
@@ -0,0 +1,15 @@
+use std::io;
+
+use clap::{CommandFactory, Parser};
+use clap_complete::{generate, Shell};
+
+#[derive(Parser)]
+pub struct Args {
+    #[clap(value_enum, help = "The shell to generate completions script for")]
+    pub shell: Shell,
+}
+
+pub fn command(args: &Args) {
+    let mut app = Args::command();
```

Comment:
This is getting the current command `todo completion` and suggesting for that command irrespective of which command I am in.